hpy/tools/autogen/public_api.h

```diff
@@ -406,7 +406,7 @@ int HPy_TypeCheck(HPyContext *ctx, HPy obj, HPy type);
  * Return the type's name.
  *
  * Equivalent to getting the type's ``__name__`` attribute. If you want to
- * retrieve the type's name as Python unicode object, then just use
+ * retrieve the type's name as Python Unicode object, then just use
```

Comment:
PyUnicodeObject?

Reply:
I wouldn't use `PyUnicodeObject` because this type does not exist in HPy. Should I write `Python string`?

Reply:
I don't know. We need to have a terminology that makes sense, this is probably not the only place where we need this kind of construct. It is not a python `str` object since those do not exist in C, neither is it a `PyUnicodeObject`, so what is it?

Reply:
IMO, it is a handle that refers to a Python string. The [Python doc](https://docs.python.org/3/library/stdtypes.html#textseq) calls it: _str object_.

Reply:
So "retrieve the type's name as a handle that refers to a `str`, then just use..."

It should be clear that handles refer to python objects.